src/lib.rs

```diff
@@ -12,15 +12,11 @@ use cairo_lang_filesystem::db::FilesGroup;
 use cairo_lang_filesystem::ids::FileId;
 use cairo_lang_semantic::{diagnostic::SemanticDiagnosticKind, SemanticDiagnostic};
 use cairo_lang_utils::Upcast;
-use serde::{Deserialize, Serialize};
 
 pub static CAIRO_LINT_TOOL_NAME: &str = "cairo-lint";
 
 /// Describes tool metadata for the Cairo lint.
-#[derive(Serialize, Deserialize, Clone, Default, Debug)]
-pub struct CairoLintToolMetadata {
-    pub nopanic: bool,
-}
+pub type CairoLintToolMetadata = HashMap<String, bool>;
```

Comment:
Just to make sure: 
This is backwards compatible, right?

Reply:
Perhaps we should version this somehow, or at least have a warning that changing this in a non-backwards compatible way is prohibited (or discouraged, or having to be documented or whatever we choose is the right way to go in a specific case)